app/controllers/catalog_controller.rb

```diff
@@ -87,7 +87,7 @@ def self.modified_field
     config.add_facet_field 'based_near_label_sim', limit: 5
     config.add_facet_field 'publisher_sim', limit: 5
     config.add_facet_field 'file_format_sim', limit: 5
-    config.add_facet_field 'contributing_library_sim', limit: 5
+    config.add_facet_field 'contributing_library_sim', label: 'Contributing Library', limit: 5
```

Comment:
Did you try making this change in the locales file?

Reply:
I didn't, I will now though!

Reply:
Done! https://github.com/scientist-softserv/palni-palci/pull/685/commits/19a99e415d36c0f7afb977fbad7ef76cd8e30bb3